### DISCUSSION TITLE
Dependabot PRs

### DISCUSSION THREAD

**Discussion by naftalibeder** [rnmapbox/maps#1890] [General]
@mfazekas how do you want to handle the automatic PRs from dependabot? They kind of just clutter up the list.

I lean towards removing the integration, and doing dependency updates manually. Thoughts?

**Comment by mfazekas** [comment]
@naftalibeder I don't have strong opinion about dependabot, I don't think I've ever merged the PRs, so I guess we can disable it for now. But if someone would like to use it I don't mind that either.

**Reply by naftalibeder** [reply]
Cool - I would prefer to disable it, but since I don’t have admin level permissions, I’m unable to do so. If you did, I wouldn’t object!
